### DISCUSSION TITLE
GCP Composer worker resources

### DISCUSSION THREAD

**Discussion by julesmga** [TobikoData/sqlmesh#3363] [Q&A]
Hi all, to anyone deploying sqlmesh to GCP composer, do you have some recommendation regarding resource allocation?

Our workers keep restarting during plan DAGs execution, there is no error log and memory usage is below 50% so this is quite mysterious. We tried increasing memory to 3G but it didn't help.

**Comment by cmgoffena13** [comment]
@julesmga -- a little late here -- did you end up resolving this? If not, could you share approximately how many models you have in your project? That would help determine the resources needed.
